apps/web/src/utils/convertDetailToForm.ts

```diff
@@ -1,18 +1,23 @@
-import type { RecruitmentDetailDto, TextQuestionDto, FileQuestionDto } from "@web/types/recruitment";
-import type { FormValues } from "@web/types/application";
+import type {
+  RecruitmentDetailDto,
+  TextQuestionDto,
+  FileQuestionDto,
+} from '@web/types/recruitment';
+import type { FormValues } from '@web/types/application';
+import { normalizeDateStr } from './convertFormToRequest';
 
 const DRAFT_FUTURE_DATE = '2027-05-30';
 
-
 export function convertDetailToForm(detail: RecruitmentDetailDto): FormValues {
-  const interviewDuration: FormValues["interviewDuration"] =
+  
+  const interviewDuration: FormValues['interviewDuration'] =
```

Comment:
이 부분 객체 매핑을 사용해서 처리하면 더 가독성에 좋을거 같아요!
```suggestion
const DURATION_MAP: Record<number, FormValues['interviewDuration']> = {
  15: '15분',
  30: '30분',
  60: '1시간',
};
const interviewDuration =
  DURATION_MAP[detail.interviewDuration] ?? '30분';
```